swagger.yaml

```diff
@@ -1113,6 +1113,10 @@ paths:
           in: path
           required: true
           description: Report type [weekly, monthly, quarterly].
+        - name: period_to
+          type: string
+          in: query
+          description: Specify end of time period for the report (in ISO-8601 format)
```

Comment:
maybe an example would be useful here

Reply:
also it looks like we assume it to be in UTC format and ignore time zone, am I correct? if so, we should specify it in here